edxval/api.py

```diff
@@ -284,18 +284,17 @@ def get_video_transcript_data(video_id, language_code):
 
     Returns:
         A dict containing transcript file name and its content.
+    Raises:
+        Raises TranscriptsGenerationException if the trasncript data cannot be read.
     """
     video_transcript = VideoTranscript.get_or_none(video_id, language_code)
+    message = ('[edx-val] Error while retrieving transcript for video=%s -- language_code=%s', video_id, language_code)
```

Comment:
I think this would make more sense to be defined inside the except block as that is where it is used. It doesn't hurt anything to have it here, but it makes more sense to see it defined in the context of the exception handling.